### ISSUE TITLE
Reimporting a project a second time causes an "internal_error"

### ISSUE THREAD

**Issue by TechSupport26** [lightningrodlabs/acorn#243]
v4.0.0-alpha
Windows 10 Pro

To explain in more detail I already had this project imported but I did it a second time causing all the projects that I had imported to not load. I can't import projects and I think that it is due to the attached image below. (I was hoping that it would reset the project to that state (But thinking on that it would be really bad if anyone could do that in a group project if they had malicious intent but it could be good to restore to a previous version of the project) or to have a second copy of the project as I accidentally deleted a card and I have no idea as to what it was.)

I also can't import any projects now (I get stuck on the importing project splash text). I can create new projects but when I go back to the main menu it disappears as I think it tries to load it again in the bundle.

### Edit:
I just tried joining a project using the secret phrase and it comes up, I can open it and the changes I make are saved. (This does not apply to all project phrases, some of them did not load and would crash acorn unless you cancelled syncing them. You can also avoid this if you open Acorn in a terminal and do ctrl-c as it did not close acorn but cancelled something else that it was doing allowing acorn to refresh) End of edit.

"Attempted to commit a bundle to the source chain, but the source chain head has moved since the bundle began. Bundle head: Some(ActionHash(uhCkkJWYvi266vLDjCOZWrwTT4gLtzLAga-AlAiuRi1M52VdHcM_t)), Current head: Some((ActionHash(uhCkkwrsBKT15oJw0D354K1dY7XhK44BXIliYzU-3IiIcBsCJRkQl), 1119, Timestamp(2023-01-10T19:53:30.345183Z)))"
![image](https://user-images.githubusercontent.com/65631811/211650503-3768e46d-d080-4a84-bc9a-d4818db17bed.png)

```
PS C:\Users\zephd\AppData\Local\Programs\acorn> start Acorn
PS C:\Users\zephd\AppData\Local\Programs\acorn>
holochain > Looking for passphrase piped to stdin
holochain > Found passphrase, continuing...
holochain stderr err > WARNING: running without local db encryption

an unhandled error occurred: Error: WARNING: running without local db encryption

    at Socket.<anonymous> (C:\Users\zephd\AppData\Local\Programs\acorn\resources\app.asar\node_modules\@lightningrodlabs\electron-holochain\dist\src\holochain.js:119:33)
    at Socket.emit (node:events:527:28)
    at addChunk (node:internal/streams/readable:315:12)
    at readableAddChunk (node:internal/streams/readable:289:9)
    at Readable.push (node:internal/streams/readable:228:10)
    at Pipe.onStreamRead (node:internal/stream_base_commons:190:23)
holochain > WARNING: running without local db encryption
holochain > 1
holochain > # lair-keystore connection_url # named-pipe:\\.\pipe\24w39KvTAyKpJiY5d34lg?k=fJwfC-tVlWfbCamzM5SSGM9W8xnkvVijVqRDm0nsRHI #
holochain > # lair-keystore running #
holochain >
holochain > ###HOLOCHAIN_SETUP###
holochain > ###ADMIN_PORT:1235###
holochain > ###HOLOCHAIN_SETUP_END###
holochain > DATASTORE_PATH: C:\Users\zephd\AppData\Roaming\acorn\databases-6
holochain > KEYSTORE_PATH: Some("C:\\Users\\zephd\\AppData\\Roaming\\acorn\\keystore-3")
holochain > NETWORK_SEED: None
holochain > Recognized a keypair, using that...
holochain > An existing configuration and identity was found, using that.
holochain >      APP_WS_PORT: 8889
holochain > INSTALLED_APP_ID: acorn-project-131697-uid-dating-automatic-dial-specimen-plating
holochain > HOLOCHAIN_RUNNER_IS_READY
holochain > 7
received checkForMigrationData
```

I don't know if there is a log file or where it is but if so then if someone could message the directory path to where it usually is that would be great.

TL;DR If you have your project secret code then you can access your projects. If not then you can find them in the conductor file in C:\Users\Username\AppData\Roaming\acorn\databases-6\conductor I don't know about Mac OS or Linux

**Comment by Connoropolous** [comment]
there's a lot going on here, which I will come back to, but can I ask that you just summarize for me:
do you have currently have access to the important projects to you, or the ability to restore them, or are you blocked from both of those things?

**Comment by TechSupport26** [comment]
I can access almost all of the projects (They are the important ones). The three that I can't load are not important and I did not make any changes to the projects yet so resetting and starting from scratch is on the table.

When I join a project to access them they will come up but disappear when I leave the project or refresh.

I have the ability to restore projects by putting the secret code in.


Just something that I have noticed is that sometimes when I transfer to a new release of Acorn some of the projects lose their project cover image.

Thanks.

**Comment by TechSupport26** [comment]
This is fixed with version 8.0.0
